server/src/main/java/org/opensearch/action/get/MultiGetResponse.java

```diff
@@ -63,6 +63,10 @@ public class MultiGetResponse extends ActionResponse implements Iterable<MultiGe
     private static final ParseField ID = new ParseField("_id");
     private static final ParseField ERROR = new ParseField("error");
     private static final ParseField DOCS = new ParseField("docs");
+    // In mixed clusters, the 1.x cluster could still return the '_type' in the response payload, it has to
+    // be handled gracefully
+    @Deprecated(forRemoval = true)
+    private static final ParseField TYPE = new ParseField("_type");
```

Comment:
@nknize no objections for this change from your side? thank you!

Reply:
no objections since it's a compatibility issue! Especially w/ the clear deprecation. Nice!